### DISCUSSION TITLE
Jacdac MSR218 base schematics

### DISCUSSION THREAD

**Discussion by vorillaz** [microsoft/jacdac#1369] [Q&A] [answered]
Hi there, 
I've just stumbled upon MSR218 base as a really nice expansion board for XIAO microcontrollers.
Is there any chance you could publish the schematics and the PCB designs?

I've looked around with no success so far.

Thanks in advance.

**Comment by mmoskal** [comment, marked as answer]
See attached. Note that it was not EMC tested or even looked at by an EE (though it is quite simple)

[msr218.zip](https://github.com/microsoft/jacdac/files/14968349/msr218.zip)

**Reply by vorillaz** [reply]
That's amazing, thank you so much. Just gathering references for prototyping a XIAO board actually so this one would be really helpful.

**Reply by pelikhan** [reply]
Are building a new jacdac board?

**Reply by vorillaz** [reply]
I am not so good at schematics, basically trying to grab some inspiration because I am experimenting with XIAO ESP32C3 and Deviscript.
